pipelines/shopgate.user.getFavorites.v1.json

```diff
@@ -109,35 +142,64 @@
         "id": "@shopgate/favorites",
         "path": "@shopgate/favorites/sortItems.js",
         "input": [
-          {"key": "productIds", "id": "120"},
-          {"key": "products", "id": "130"}
+          {
+            "key": "productIds",
+            "id": "120"
+          },
+          {
+            "key": "products",
+            "id": "130"
+          }
         ],
         "output": [
-          {"key": "products", "id": "130"}
+          {
+            "key": "products",
+            "id": "130"
+          }
         ]
       },
       {
         "type": "extension",
         "id": "@shopgate/magento-cart",
         "path": "@shopgate/magento-cart/products/requestParentProductsFromMagento.js",
         "input": [
-          {"key": "products", "id": "130"},
-          {"key": "token", "id": "140"}
+          {
+            "key": "products",
+            "id": "130"
+          },
+          {
+            "key": "token",
+            "id": "140"
+          }
         ],
         "output": [
-          {"key": "magentoParentProducts", "id": "150", "optional": true}
+          {
+            "key": "magentoParentProducts",
+            "id": "150",
+            "optional": true
+          }
         ]
       },
       {
         "type": "extension",
         "id": "@shopgate/magento-cart",
         "path": "@shopgate/magento-cart/products/appendProductVariantsMetadataToProductList.js",
         "input": [
-          {"key": "products", "id": "130"},
-          {"key": "magentoParentProducts", "id": "150", "optional": true}
+          {
```

Comment:
even though I like them unwrapped, we had been using the practice of wrapping for quick readability of what the pipeline does. It seems like other teams are also doing it too. So meh...